examples/shared_vpc_workspaces/0_workspace_data_lookup/main.tf

```diff
@@ -0,0 +1,42 @@
+terraform {
+  required_providers {
+    aptible = {
+      source  = "aptible.com/aptible/aptible-iaas"
+    }
+  }
+}
+
+variable "aptible_host" {
+  type    = string
+}
+
+variable "organization_id" {
+  type    = string
+}
+
+variable "environment_id" {
+  type    = string
+}
+
+variable "vpc_id" {
+  type = string
+}
+
+provider "aptible" {
+  host = var.aptible_host
+}
+
+data "aptible_organization" "org" {
+  id = var.organization_id
+}
+
+data "aptible_environment" "env" {
+  id      = var.environment_id
+  org_id  = data.aptible_organization.org.id
+}
+
+data "aptible_aws_vpc" "main" {
```

Comment:
Using a data source is not ideal because then it relies on another workspace to manage the resource.  It seems like what we need is the ability in our `Read` for a VPC resource is to be able to look up an asset by `name` instead of just `id`.  Then when we run `terraform apply` it'll check to see if the resource already exists.  Does that make sense?

Reply:
i think looking up by name is better

Reply:
When searching by name and does not exist `asdf`

<img width="752" alt="image" src="https://user-images.githubusercontent.com/2961973/197224309-5cac054f-12dd-4e84-b2c3-047a37d565cd.png">

When searching by name and does exist:

<img width="1190" alt="image" src="https://user-images.githubusercontent.com/2961973/197225264-9d74d16d-b667-457d-9235-f569a55cc928.png">